criu/cr-dump.c

```diff
@@ -1,52 +1,18 @@
-#include <stdio.h>
-#include <stdlib.h>
-#include <stdarg.h>
-#include <signal.h>
-#include <limits.h>
-#include <unistd.h>
-#include <errno.h>
-#include <string.h>
-
-#include <fcntl.h>
-
-#include <sys/types.h>
-#include <sys/stat.h>
-#include <sys/vfs.h>
-#include <sys/time.h>
-#include <sys/wait.h>
-
-
-#include <sched.h>
```

Comment:
Oh wow, I **didn't** mean to ask you to remove all the unused headers in all the existing files =). (Though it might be a matter of another patch-set.)

I mean to remove all unused headers **you add** in you patch-set **only**.

Also you should merge fixes to your original commits into your original commits. In the patch-set each commit should be "good" there is no such a situation where first commit is breaking something and second commit is fixing (cleanup-ing as in your case) the first one.

Reply:
Oh, sorry. I was not aware of these guidelines, though I have learned a lot in the past few days. Thanks a lot for your inputs. 
Just to clarify, should I restore the headers which I have removed or let the file be as it is now?
Also, should I revert to the previous three commits and send a fresh 'good' commit?

Reply:
1) Let's restore. So If your patchset as a whole **adds** some new headers then just check if they (new ones) are really required.
2) You can merge your original commit and all fixup changes together in a single fresh commit in the same branch an push it with force "-f" to replace commits in this pull-request.

Glad to help!